packages/blockchain/src/blockchain.ts

```diff
@@ -352,7 +355,7 @@ export class Blockchain implements BlockchainInterface {
 
       this.DEBUG &&
         this._debug(
-          `Canonical head set from ${bytesToHex(canonicalHeadHash)} to ${bytesToHex(hash!)} (number ${bigIntToHex(canonicalHead)})`,
+          `Canonical head set from ${canonicalHeadHash} to ${bytesToHex(hash!)} (number ${bigIntToHex(canonicalHead)})`,
```

Comment:
Should the `bytesToHex` be removed here?

Reply:
No, since it's going to be bytes. Fixed.